Makefile

```diff
@@ -112,10 +116,10 @@ else
   OPTFLAGS := -O2
 endif
 
-ASFLAGS := -march=vr4300 -32 -Iinclude
+ASFLAGS := -march=vr4300 -32 -no-pad-sections -I include
```

Comment:
The handwritten `gu` functions are aligned to 32 (icache line), and without this flag `as` will pad with too many nops after the function.

Reply:
I see, I didn't know it would add more padding because of that.

An alternative would be to add this flag specifically for `gu` files (which I expect would be in a dedicated folder eventually), but I don't know if padding sections is really important anyway

Reply:
After this PR is over with I'd like to make another that moves them into src/ folders, the original libultra directory structure (visible in mdebug) has the handwritten gu functions in a folder called mgu, so we could restrict it at that point.
I don't believe this option should affect anything at least in the short term, maybe when we need GLOBAL_ASM again but by then we probably will need to sort out a bunch of other stuff too.

Reply:
I was more thinking along the lines of modding or portability for situations where padding could matter, but I don't know if it's worth worrying about since this is handwritten asm and not really subject to change

Reply:
Ah for modding I'm pretty sure it shouldn't matter at all. The linker should be able to resolve any problems that might arise with respect to alignment or padding